src/orchestrator/orchestrator.rs

```diff
@@ -563,9 +563,9 @@ impl<'a> Drop for JobTask<'a> {
             // If there are dependencies, the error is probably from another task
             // If there are no dependencies, the error was caused by something else
             let errmsg = if self.jobdef.dependencies.is_empty() {
-                "error occurred"
+                "error occured".red()
```

Comment:
Why change "occurred" to "occured"?
https://writer.com/blog/occurred-occured/
(I (hopefully) fixed that as part of 53de218d1941c3ecd45c623e7570e9eb8ff3d83c...)